guides/release/components/built-in-components.md

```diff
@@ -108,24 +107,24 @@ Recall that there were a few exceptions. The following input attributes must be
 
 ### Actions
 
-Starting with Ember Octane, we recommend using the `{{on}}` modifier to call an action on specific events such as the [input event](https://developer.mozilla.org/docs/Web/API/HTMLElement/input_event).
+À partir de Ember Octane, il est recommandé d'utiliser le modifier `{{on}}` pour appeler une action lors d'un _event_ (événement se produisant sur la page) comment [l'event "input"](https://developer.mozilla.org/fr/docs/Web/API/HTMLElement/input_event).
 
 ```handlebars
-<label for="input-name">Name:</label>
+<label for="input-name">Nom :</label>
 <Input
   id="input-name"
   @value={{this.name}}
   {{on "input" this.validateName}}
 />
 ```
 
-[Learn more about the `{{on}}` modifier.](../../upgrading/current-edition/action-on-and-fn/#toc_the-on-modifier)
+[À propos du modifier `{{on}}`](../../upgrading/current-edition/action-on-and-fn/#toc_the-on-modifier)
 
-Lastly, Ember also provides custom input events `@enter`, `@insert-newline` and `@escape-press`. These events do not exist on native input elements, but you may find them to be useful for handling keyboard interactions.
+Pour terminer, Ember fournit également plusieurs events d'input spécifiques :  `@enter`, `@insert-newline` et `@escape-press`. Ces events n'existent pas sur les éléments input natifs, mais vous pourriez les trouver utiles pour implémenter les intéractions clavier.
 
-The modern, Octane-style way to handle keyboard events is to [write a modifier](../../upgrading/current-edition/glimmer-components/#toc_writing-your-own-modifiers) to separate concerns: The component manages the state, while the modifier manages interactions with the DOM. Your action will receive an actual `event` object.
+Le style Octane moderne pour gérer les events clavier est [d'écrire un modifier](../../upgrading/current-edition/glimmer-components/#toc_writing-your-own-modifiers) afin de séparer les responsabilités : le composant gère les états, tandis que le modifier gère les intéractions avec le DOM. L'action appelée lors d'un event reçoit alors un objet `event` en paramètre.
```

Comment:
I'm not sure about 
> Le style Octane moderne pour gérer

What is modern, is the Octane style.

Maybe this is a little bit better : 

```suggestion
Le style moderne (ou Ember Octane) pour gérer les events clavier est [d'écrire un modifier](../../upgrading/current-edition/glimmer-components/#toc_writing-your-own-modifiers) afin de séparer les responsabilités : le composant gère les états, tandis que le modifier gère les intéractions avec le DOM. L'action appelée lors d'un event reçoit alors un objet `event` en paramètre.
```